felix/ip/trie.go

```diff
@@ -325,31 +325,56 @@ func (t *CIDRTrie) Visit(f func(cidr CIDR, data interface{}) bool) {
 	t.root.visit(f)
 }
 
-// Descendants returns a list of CIDRs representing the closest descentents of the given CIDR in the trie that
-// have data associated with them.
-func (t *CIDRTrie) Descendants(parent CIDR) []CIDR {
-	// Get the node representing this CIDR. We include intermediate nodes so that
-	// we can traverse them to find the closest descendants.
+// ClosestDescendants returns a list of CIDRs representing the closest descendants of the given CIDR in the trie that
+// have data associated with them. It does not return a full list of all descendants - only any descendants that are tied for
+// being the closest to the given CIDR.
+//
+// For example, given the following trie, where * indicates an intermediate node with no data associated with it:
+//
+//	  		  10.0.0.0/16
```

Comment:
This looked a lot better in my editor than it does here :sweat_smile:

Reply:
Looks like it's got mixed tabs and spaces